packages/manager/modules/web-universe-components/src/mxplan-banner/mxplan-banner.component.js

```diff
@@ -0,0 +1,8 @@
+import template from './mxplan-banner.html';
+import controller from './mxplan-banner.controller';
+
+export default {
+  name: 'mxplanBanner',
```

Comment:
```suggestion
  name: 'wucMxplanBanner',
```

So we know the component is on Web Universe Component